action.yml

```diff
@@ -9,35 +9,38 @@ inputs:
     description: URL pointing to your Mayhem instance
     required: false
   owner:
-    description: sets the owner of the project (organization or user).
+    description: Sets the owner of the project (workspace or account name).
     required: false
     default: ${{ github.repository_owner }}
+  project:
```

Comment:
Could you used default instead of doing it programmatically in the JS?

Reply:
> Defaults to the name of your repository if not defined.

`default: ${{github.event.repository.name}}`

Reply:
May not work in all scenarios though...
https://stackoverflow.com/questions/62803531/repository-name-as-a-github-action-environment-variable

Reply:
```
const repo = process.env["GITHUB_REPOSITORY"];
    if (repo === undefined) {
      throw Error(
        "Missing GITHUB_REPOSITORY environment variable. " +
          "Are you not running this in a Github Action environment?"
       );
     }
```

I guess if we had this in the JS, it probably wasn't possible with defaults in the action.yml

Reply:
👆 right, this is basically the same path I went through :)